src/components/Modal/index.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useEffect } from "react";
+import ReactDOM from "react-dom";
+import { IconClose } from "@stellar/design-system";
+
+import "./styles.scss";
+
+const MODAL_OPEN_CLASS_NAME = "modal-open";
+
+interface ModalProps {
+  visible: boolean;
+  onClose: () => void;
+  children: React.ReactNode;
+  disableWindowScrollWhenOpened?: boolean;
+}
+
+export const Modal = ({
```

Comment:
With HTML forms, the `<button type="submit"...` tag allows you to press Enter/Return and it will submit the form. Try it out with the SRT SEP-24 flow. 

I don't think we'd want to pre-select the submit button since they need to fill out the form, but can we not have an event listener for the enter key that call the same function that the button's `onClick` property would call?

```
      <div className="ModalButtonsFooter">
        {isPending && <Loader />}

        <Button
          onClick={handleSetUntrustedAsset}
          disabled={!(assetCode && homeDomain) || isPending}
        >
          Add
        </Button>
```

For example, we could have an event listener for the Enter key that checks the conditions outlined in `disabled` and calls `handleSetUntrustedAsset` if the conditions are `true`.

Reply:
It is a bit tricky because modal will be a component in the Design System. I'll note this and will try to come up with a solution inside the modal component so that we don't need to do it manually every time. Maybe a prop to enable that feature in the modal. 🤔 It would be a cool feature to have.

Reply:
Created an issue https://github.com/stellar/nebula/issues/163, so that we don't forget about it.